### ISSUE TITLE
Spot immediately shuts down after login.

### ISSUE THREAD

**Issue by jakubiszon26** [xou816/spot#448]
**Describe the bug**
I was able to sign in to the app but immediately after it crashed
**To Reproduce**
1. Login or open app after login
2. See how it crashes
**Expected behavior**
Playing wonderfull death metal
**Screenshots**

https://user-images.githubusercontent.com/61655602/150698288-e5a92f01-4eca-4e24-8811-82924fe4d264.mp4

**General information:**
 - Distribution: Fedora Workstation 35
 - Installation method [e. g. built from source, installed from Flathub...]: Flathub
 - Version [e.g. 0.1.0]: Flathub latest (0.3.0)
 - Device used [e. g. desktop, phone...]: Laptop (huawei matebook 13)

**Stack trace:**
```
thread 'main' panicked at 'Argument count has to match number of format directives ({})', src/app/components/labels.rs:23:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```
```
flatpak run --env=RUST_BACKTRACE=full dev.alextren.Spot
```
```thread 'main' panicked at 'Argument count has to match number of format directives ({})', src/app/components/labels.rs:23:5
stack backtrace:
   0:     0x56052eba8eac - <unknown>
   1:     0x56052ebcdadc - <unknown>
   2:     0x56052eba12f5 - <unknown>
   3:     0x56052ebaaad0 - <unknown>
   4:     0x56052ebaa685 - <unknown>
   5:     0x56052ebab184 - <unknown>
   6:     0x56052ebaac60 - <unknown>
   7:     0x56052eba9354 - <unknown>
   8:     0x56052ebaabc9 - <unknown>
   9:     0x56052e2419d1 - <unknown>
  10:     0x56052ebcb9c1 - <unknown>
  11:     0x56052e2418cb - <unknown>
  12:     0x56052e4a24fd - <unknown>
  13:     0x56052e3d4e7b - <unknown>
  14:     0x56052e4a081b - <unknown>
  15:     0x56052e2f102a - <unknown>
  16:     0x56052e477fad - <unknown>
  17:     0x56052eb84027 - <unknown>
  18:     0x56052eb7fcca - <unknown>
  19:     0x7ff3afe256d1 - g_main_context_dispatch
  20:     0x7ff3afe25bb8 - <unknown>
  21:     0x7ff3afe25c83 - g_main_context_iteration
  22:     0x7ff3b0058c45 - g_application_run
  23:     0x56052e344d7e - <unknown>
  24:     0x56052e3c9cb3 - <unknown>
  25:     0x7ff3afe26a15 - g_main_context_invoke_full
  26:     0x56052e30cfb1 - <unknown>
  27:     0x56052e2f6d03 - <unknown>
  28:     0x56052e3ebdd9 - <unknown>
  29:     0x56052eba8b8b - <unknown>
  30:     0x56052e30d632 - main
  31:     0x7ff3afa8bb90 - __libc_start_main
  32:     0x56052e2421fe - <unknown>
  33:                0x0 - <unknown>
```

**Comment by xou816** [comment]
Hi! Thanks for the report! This one's on me, I messed up some translation strings, and there will be this crash for a couple of locales :/ you can force the use of the english locale in the meantime, but I'll work on a patch asap

**Comment by xou816** [comment]
Should be fixed in 0.3.1
